mandatory/4-tax.js

```diff
@@ -5,8 +5,17 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
-
+function calculateSalesTax(price) {
+  let totalPrice = price * 0.2 + price;
+  return totalPrice;
+}
+let totalPrice = calculateSalesTax(15);
+console.log(totalPrice);
+let totalPrice2 = calculateSalesTax(17.50);
+console.log(totalPrice2);
+let totalPrice3 = calculateSalesTax(34);
+console.log(totalPrice3);
```

Comment:
We don't want to keep any debugging code in our tests. We should remove all the `console.log`s from our test files.
The tests furthur in the file will correctly evaluate the tests.